src/cart/tests_common.h

```diff
@@ -104,6 +108,35 @@ tc_progress_stop(void)
 	opts.shutdown = 1;
 }
 
+/* Write a completion file to signal graceful server shutdown */
+void
+write_completion_file(void)
+{
+	FILE	*fptr;
+	char	*dir;
+	char	*completion_file;
+	char	*tmp_str;
+	char	pid[6];
+	pid_t	_pid;
+
+	_pid = getpid();
+	sprintf(pid, "%d", _pid);
+
+	dir = getenv("DAOS_TEST_SHARED_DIR");
```

Comment:
what if dir is null?

Reply:
I think we should assert non-null on `dir`.  In other words, we require `DAOS_TEST_SHARED_DIR` be set for using the `--write_completion_file` option.  Unless there are alternative mounted directories (which avocado can access) to write completion files.

Reply:
sounds good, lets do it

Reply:
984ab8326148b1094ecf16bd2e180f90ebf80c23